client/my-sites/google-my-business/stats/chart.js

```diff
@@ -193,7 +199,9 @@ class GoogleMyBusinessStatsChart extends Component {
 					<LineChart
 						fillArea
 						data={ transformedData }
-						renderTooltipForDatanum={ renderTooltipForDatanum }
+						renderTooltipForDatanum={
+							renderTooltipForDatanum ? renderTooltipForDatanum( interval ) : undefined
```

Comment:
nitpick: On each render, we pass a new function to `LineChart`'s `renderTooltipForDatanum`'s prop,
because `renderTooltipForDatanum( interval )` returns a new function each time